tests/Tokenizer/TransformersTest.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Tests\Tokenizer;
+
+use PhpCsFixer\Test\AccessibleObject;
+use PhpCsFixer\Tokenizer\Transformers;
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Finder\Finder;
+
+/**
+ * @author SpacePossum
+ *
+ * @internal
+ *
+ * @covers \PhpCsFixer\Tokenizer\Transformers
+ */
+final class TransformersTest extends TestCase
+{
+    public function testCreateOneTime()
+    {
+        $transformers = Transformers::create();
+        $this->assertSame($transformers, Transformers::create());
+    }
+
+    public function testTransform()
+    {
+        $transformers = Transformers::create();
+        $accessible = new AccessibleObject($transformers);
+        $transformers = $accessible->items;
+
+        $this->assertInternalType('array', $transformers);
+        $this->assertContainsOnlyInstancesOf('PhpCsFixer\Tokenizer\TransformerInterface', $transformers);
+
+        $classes = [];
+        /** @var \SplFileInfo $file */
+        foreach (Finder::create()->files()->in(__DIR__.'/../../src/Tokenizer/Transformer') as $file) {
+            $class = '\\PhpCsFixer\\Tokenizer\\Transformer\\'.$file->getBasename('.php');
+            $this->assertTrue(class_exists($class, false), sprintf('Failed transformer "%s" is loaded by Transformers', $class));
```

Comment:
This assertion is weak IMO: it will pass even if the class was loaded via some unrelated code, which does not mean an actual instance of the class is in the `items` array. I think it would be better to check that the array contains an instance of the class instead.